src/main/java/ProjectIndexer.java

```diff
@@ -42,7 +42,7 @@ public void index() {
         spoon.getEnvironment().setIgnoreDuplicateDeclarations(true);
         spoon.getEnvironment().setComplianceLevel(9);
         spoon.buildModel();
-        Map<String, CtElement> typeByFile = new HashMap();
+        Map<String, CtElement> typeByFile = new HashMap<>();
```

Comment:
The real change 😛

Reply:
:brain: officially production ready :rocket: